crates/core/tests/integration_tests.rs

```diff
@@ -57,7 +81,9 @@ fn check_insert(conn: &Connection, sql: &str, params: Params) {
     conn.execute(sql, params).unwrap();
     let rows = conn.execute("SELECT * FROM users", ()).unwrap().unwrap();
     let row = rows.next().unwrap().unwrap();
-    assert_eq!(row.get::<i32>(0).unwrap(), 1);
+    // Use two since if you forget to insert an id it will automatically
+    // be set to 1 which defeats the purpose of checking it here.
+    assert_eq!(row.get::<i32>(0).unwrap(), 2);
```

Comment:
I made this change because I was really confused when my test was not failing when I was not inserting the first column (It gets auto added via primary key in the schema). So I changed this to ensure we capture any missed params.